### PR TITLE
Extract menuitems from profile

### PR DESCRIPTION
**Pull request by Charokee**

### CODE REVIEW

File: ui/src/js/profile/ReceiptsPage.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      view: 'receipts'
+    };
+  }
+
+  switchTo(value) {
+    this.setState({view: value})
+  }
+
+  render() {
+    var content;
```
Comment (mschaaf) [inline ui/src/js/profile/ReceiptsPage.js]: Please set a default value to clarify what we expect in of no receipts. Thanks.

File: ui/src/js/profile/ToolsPage.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    var toolsPart;
+    var page = this.state.pageCount;
+
+    if (this.state.user.editAllowed) {
+      toolsPart = <Tools/>;
+    } else {
+      toolsPart = '';
+    }
+
+    toolsPart = <Tools/>;
```
Comment (mschaaf) [inline ui/src/js/profile/ToolsPage.js]: This overrides always the set value from line 40 to 44. Maybe this is wrong placed or the lines from 40 to 44 are not necessary.

File: ui/src/js/profile/ReceiptsPage.js (hunk 1 above)
Comment (Charokee) [reply ui/src/js/profile/ReceiptsPage.js]: Added default contnt

File: ui/src/js/profile/ToolsPage.js (hunk 2 above)
Comment (Charokee) [reply ui/src/js/profile/ToolsPage.js]: Removed unnecessary code

**Comment by Charokee** [comment]
Hello Team! Are the requested changes ready to merge?

**Comment by mschaaf** [comment]
@Gab0rB from my side the changes are good please also have a look.

File: ui/src/js/profile/ToolsPage.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  callPreviousPage() {
+    var newPage = this.state.pageCount - 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
```
Comment (Gab0rB) [inline ui/src/js/profile/ToolsPage.js]: i think these methods(callNext- and callPrevoiusPage) are related to the planted tree requests on the profile page,
so they can be removed here.

File: ui/src/js/profile/ToolsPage.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  updateLanguage(value) {
+    this.refs['navbar'].updateLanguage(value);
+  }
+
```
Comment (Gab0rB) [inline ui/src/js/profile/ToolsPage.js]: this method seems to be a copy&paste mistake, i don't see any option to call this.

**Comment by Gab0rB** [comment]
I added also 2 comments, please have a look.

File: ui/src/js/profile/ToolsPage.js (hunk 3 above)
Comment (Charokee) [reply ui/src/js/profile/ToolsPage.js]: Yes, I remember, we talked about it before my holidays. I removed the methods.

File: ui/src/js/profile/ToolsPage.js (hunk 4 above)
Comment (Charokee) [reply ui/src/js/profile/ToolsPage.js]: You are right, there is no functionality to switch the language in the component.

**Comment by Charokee** [comment]
Thanks for the remarks and corrections! I think we are now ready to merge the branch.

**Pull request merged**